CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 As a minor extension, we also keep a semantic version for the `UNRELEASED`
 changes.
 
+## Unreleased
+
+- Tested with `cardano-node 8.11.0` and `cardano-cli 8.23.1.0`.
```

Comment:
```suggestion
- **DO NOT RELEASE** as the tested `cardano-node` version is not intended to be used on `mainnet` yet.
- Tested with `cardano-node 8.11.0-pre` and `cardano-cli 8.23.1.0` on `sanchonet` ONLY.
```

Must: ensure we do not release this version as long as `mainnet` is not supported by the `cardano-node` version we suggest using (e.g. in tutorials etc.)